x/gov/abci.go

```diff
@@ -19,7 +19,7 @@ func EndBlocker(ctx sdk.Context, keeper keeper.Keeper) {
 	// delete inactive proposal from store and its deposits
 	keeper.IterateInactiveProposalsQueue(ctx, ctx.BlockHeader().Time, func(proposal types.Proposal) bool {
 		keeper.DeleteProposal(ctx, proposal.ProposalId)
-		keeper.DeleteDeposits(ctx, proposal.ProposalId)
+		keeper.DeleteAndBurnDeposits(ctx, proposal.ProposalId)
```

Comment:
This is an go API breaking change. We should be careful to document this and changed behavior in the changelog.

Reply:
inactive proposal is a proposal which is in deposit period. So here, we want to remove dead proposals: proposals which didn't get enough deposit to get on voting phase. 

What's the correct spec for this? Should we be able to query dead proposal deposits?